main.py

```diff
@@ -84,11 +84,20 @@ def img(soup):
 
 def GetPage(path):
     # filter our target path
-    if path[0] != '/':
+    if path[0] != '/' or path in Visited_page:
         return
+    Visited_page.append(path)
     print(path)
     request = requests.get(Pycon_url + path)
```

Comment:
We cloud introduce [pathlib](https://docs.python.org/3/library/pathlib.html) here for helping us dealing with the path related operation